src/java.base/share/classes/sun/security/tools/keytool/Main.java

```diff
@@ -1118,6 +1126,7 @@ void doCommands(PrintStream out) throws Exception {
             }
         }
 
+        buildTrustedCerts();
```

Comment:
No change. This is because `caks` global variable can only be initialized with cacerts keystore  when the `trustcacerts` option is specified; otherwise if has to be kept null. `buildTrustedCerts`() is always executed.

Reply:
Done as suggested.